pkg/parse/run.go

```diff
@@ -418,37 +428,15 @@ func updateSyncStatus(ctx context.Context, p Parser) {
 			return
 
 		case <-updateTimer.C:
+			// Combine remediator and applier errors
+			syncErrs := status.Append(p.RemediatorConflictErrors(), p.ApplierErrors())
+
 			klog.V(3).Info("Updating sync status (periodic while syncing)")
-			if err := p.SetSyncStatus(ctx, p.options().updater.applier.Errors()); err != nil {
+			if err := setSyncStatus(ctx, p, state, syncErrs); err != nil {
 				klog.Warningf("failed to update sync status: %v", err)
 			}
-			remediatorErrs := p.RemediatorConflictErrors()
-			if len(remediatorErrs) > 0 {
```

Comment:
Here as well. This is to surface a similar conflict error to the other conflicting RootSync. I feel like it would be good to keep the behavior.

When the webhook is enabled, it is okay to only surface the error to the second RootSync.   But if the webhook is disabled (default behavior), two RootSync will fight with each other. It would be good to notify both RootSyncs when the conflict happens.

Reply:
Reverted this removal and moved it into `setSyncStatus`.